# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/termios.h-sparc.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

This fixes the sparc termios VMIN and VTIME to the what the Linux
kernel expects.

  - Rene Rebe <rene@exactcode.de>

diff -ur ../dietlibc-0.29-32/include/termios.h ./include/termios.h
--- ./include/termios.h	2006-06-17 21:20:11.000000000 +0000
+++ ./include/termios.h	2006-06-18 11:06:47.210000000 +0000
@@ -202,8 +202,8 @@
 #define VDISCARD 13
 #define VWERASE 14
 #define VLNEXT	15
-#define VMIN	16
-#define VTIME	17
+#define VMIN	VEOF
+#define VTIME	VEOL
 #else			/* arm, i386, parisc, s390, x86_64 */
 #define VINTR	0
 #define VQUIT	1
